e2e/features/start-appeal-out-of-country-removal-of-client.feature

```diff
@@ -0,0 +1,60 @@
+Feature: Out of country appeal removal of client
```

Comment:
"I save my initial appeal" journey to include ooc journey with check on IA_OUT_OF_COUNTRY_ENABLED